elastic/kibana
# [Alert details page][Custom threshold] Add history chart to custom threshold alert details page

Closes #175200

## Summary

This PR adds a history chart to the custom threshold alert details page. The history chart is only added if we have only 1 condition in the rule.

Also, this PR fixes the issue of not applying group by information on the main chart that I mistakenly introduced during refactoring code in this [PR](https://github.com/elastic/kibana/pull/175777).

![image](https://github.com/elastic/kibana/assets/12370520/22b449c4-71de-4714-8ab8-9fdd244eb943)

## 🧪 How to test
- Create a custom threshold rule with only one condition
- Go to the alert details page from the alert table actions
- You should be able to see the history chart for the last 30 days with the correct filtering both for optional KQL and group by information
